volume_scroller@francislavoie.github.io/extension.js

```diff
@@ -62,13 +62,23 @@ export default class VolumeScrollerExtension extends Extension {
   _handle_scroll(_actor, event) {
     let volume = this.sink.volume;
 
-    switch (event.get_scroll_direction()) {
+    let settings = new Gio.Settings({
+      schema_id: "org.gnome.desktop.peripherals.touchpad",
+    });
+    
+    let naturalScroll = settings.get_boolean("natural-scroll");
+
+    let scrollDirection = event.get_scroll_direction();
+
+    let multiplier = naturalScroll ? -1 : 1;
```

Comment:
Tiny nit, I'd group lines up like this to make the intent clear:

```suggestion
    let naturalScroll = settings.get_boolean("natural-scroll");
    let multiplier = naturalScroll ? -1 : 1;

    let scrollDirection = event.get_scroll_direction();
```